upgrade-notes/backend_20250224_112831.md

```diff
@@ -0,0 +1,5 @@
+#### Fix cart & missing whishlist/comparison products after external login ([#3822](https://github.com/shopsys/shopsys/pull/3822))
+
+- for external logins you need to make sure to properly clear the localStorage entries
+- when using `loginAsCustomerUser.html.twig` template, you need to manually perform `handleActionsAfterLogin`, mainly reset `localStorage` entries for `cartUuid`, `productListUuids`, `authLoading` and `userEntry`
+- see #project-base-diff to update your project
```

Comment:
in this PR there are no changes in the project-base, so this sentence can be removed